core-js/src/main/javascript/cdf-legacy/components/input.js

```diff
@@ -635,7 +635,9 @@ var DateInputComponent = BaseComponent.extend({
 
     myself.placeholder()
         .addClass('date-input-container')
-        .html('<input class="date-input" id="' + inputId + '" value="' + inputValue + '"/>');
+        .html('<input class="date-input" id="' + inputId + '"/>');
+
+      $('#' + inputId).val(inputValue);
```

Comment:
looks like incorrect indent here, should be 4 spaces for this line.

Reply:
Thanks!
Now it should be fixed.